src/lib/components/Subscribe/index.js

```diff
@@ -99,17 +113,11 @@ class Subscribe extends BaseElement {
 
   onSubmit(e) {
     e.preventDefault();
-    if (this.processing || this.submitted) {
+    if (this.processing || this.submitted || !this.captchaCheck()) {
```

Comment:
This will not run if a robot submits the form programatically by calling submit(). You have to overwrite that behavior separately from event listeners: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit

Reply:
Ok, so now in the `connectedCallback` I overwrite the `form.submit` to be function that logs an error asking users (bots) to fill out the form.